src/spikeinterface/core/baserecording.py

```diff
@@ -766,24 +766,21 @@ def get_num_samples(self) -> int:
 
     def get_traces(
         self,
-        start_frame: Union[int, None] = None,
-        end_frame: Union[int, None] = None,
-        channel_indices: Union[List, None] = None,
+        start_frame: int | None = None,
+        end_frame: int | None = None,
+        channel_indices: list | np.array | None = None,
```

Comment:
or tuple

Reply:
Thanks. I totally missed that one. I'll push a fix in a second.